custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -103,7 +101,9 @@ class ScenarioSimilarities(
         browseProjects: Action,
         browseFilters: Action,
         browseBoards: Action,
-        viewBoard: Action
+        viewBoard: Action,
+        projectIssueNavigator: Action
```

Comment:
Note the naming of actions. They're in *imperative mood*, that is - they sound like commands.
```suggestion
        browseProjectIssues: Action
```